src/GHC/SourceGen/Syntax/Internal.hs

```diff
@@ -103,7 +114,9 @@ builtLoc = L builtSpan
 -- In GHC-8.8, source locations for Pat aren't stored in each node, and
 -- LPat is a synonym for Pat.
```

Comment:
Update this comment on 8.10?

Reply:
Updated comment and simplified the CPP to a single if/else. (The extra logic is *only* needed for 8.8, not for 8.4, 8.6 or 8.10.)